### ISSUE TITLE
Interactive Book not loading 

### ISSUE THREAD

**Issue by adebareshowemimo** [tunapanda/h5p-standalone#125]
Uncaught TypeError: Cannot assign to read only property 'prototype' of function 'function i(t,r){var n,a=arguments.length>2&&void 0!==arguments[2]?arguments[2]:{};at(this,i);var s=ut(n=o.call(...<omitted>...n}'
    at i.newRunnable (frame.bundle.js:2:117728)
    at HTMLDivElement.<anonymous> (frame.bundle.js:2:108627)
    at Function.each (frame.bundle.js:2:15395)
    at P.fn.init.each (frame.bundle.js:2:13875)
    at i.init (frame.bundle.js:2:107649)
    at HTMLDocument.<anonymous> (frame.bundle.js:2:137900)
    at l (frame.bundle.js:2:42385)
    at u (frame.bundle.js:2:42687)

**Comment by aminemarref** [comment]
I have exactly the same problem when I try to play an interactive book using the H5P Standalone Player.

Chrome:
```
Uncaught TypeError: Cannot assign to read only property 'prototype' of function 'function i(t,r){var n,a=arguments.length>2&&void 0!==arguments[2]?arguments[2]:{};at(this,i);var s=ut(n=o.call(...<omitted>...n}'
    at i.newRunnable (frame.bundle.js:2:117728)
    at HTMLDivElement.<anonymous> (frame.bundle.js:2:108627)
    at Function.each (frame.bundle.js:2:15395)
    at P.fn.init.each (frame.bundle.js:2:13875)
    at i.init (frame.bundle.js:2:107649)
    at HTMLDocument.<anonymous> (frame.bundle.js:2:137900)
    at l (frame.bundle.js:2:42385)
    at u (frame.bundle.js:2:42687)
```

Firefox:
```
Uncaught TypeError: "prototype" is read-only
    newRunnable http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    init http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    each http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    each http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    init http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    <anonymous> http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    l http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    u http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    setTimeout handler*357/</Deferred/then/a/< http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    u http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    fireWith http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    fire http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    u http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    fireWith http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    ready http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    $ http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    357 http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    357 http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    357 http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    n http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    <anonymous> http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    <anonymous> http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    <anonymous> http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    <anonymous> http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
    <anonymous> http://localhost/tools/h5p-standalone/assets/frame.bundle.js:2
```

**Comment by aminemarref** [comment]
Ok by looking at comments on other threads, I removed `"use strict";` from `frame.bundle.js` and the interactive book is loading. Actually there are 5 occurrences of `"use strict";` in `frame.bundle.js`, so I removed the 5th occurrence which is closest to the line being reported in the error message.

**Comment by adebareshowemimo** [comment]
@aminemarref , does that solve the problem?

Many thanks

**Comment by aminemarref** [comment]
Yes it does. Just search inside the file `/assets/frame.bundle.js` for `"use strict";`, it will return 5 hits (if we are using the same file versions). Either remove all occurrences of `"use strict";`, or go the fifth occurrence of and remove it.

**Comment by adebareshowemimo** [comment]
Thanks

**Comment by murageyun** [comment]
fixed on #118
